### ISSUE TITLE
Webcamoid doesn't start because of Qt version mismatch

### ISSUE THREAD

**Issue by jkanev** [webcamoid/webcamoid#432]
## Summary

Webcamoid doesn't start because of QT version mismatch

## Current Behavior

When starting webcamoid.sh, I get the error
```
>  ./webcamoid.sh 
QQmlApplicationEngine failed to load component
qrc:/Webcamoid/share/qml/main.qml:20 plugin cannot be loaded for module "QtQuick": The plugin '/usr/lib/x86_64-linux-gnu/qt5/qml/QtQuick.2/libqtquick2plugin.so' uses incompatible Qt library. (5.15.0) [release]
```

## Expected Behavior

Webcamoid should load the plugin and start the user interface.

## Steps to Reproduce

1. Download the installer version 8.8.0, Linux 64 bit from the website
2. Install into /opt/webcamoid
3. Run /opt/webcamoid/webcamoid.sh

## Your Environment

* Webcamoid information: 8.8.0, Linux 64 bit
* Operating System information: Linux 5.10.0-6-amd64 #1 SMP Debian 5.10.28-1 (2021-04-09) x86_64 GNU/Linux
* Qt version installed: 5.15.2+dfsg-5
* File bin/qt.conf contains:
   ```
   [Paths]
   Plugins = ../lib/x86_64-linux-gnu/qt5/plugins
   Imports = ../lib/x86_64-linux-gnu/qt5/qml
   Qml2Imports = ../lib/x86_64-linux-gnu/qt5/qml
   ```

## Notes

* I don't want to downgrade my Qt5 installation. Should I compile this from source, or is there another way to run this?
* Using the version that comes with the Debian package manager (8.6.0) isn't an option either. I doesn't run, but starts with a segmentation fault.

**Comment by hipersayanX** [comment]
Is the same problem as #410 and #411. Probably a solution would be setting the rpaths in the binaries.

**Comment by ktaragorn** [comment]
Able to confirm that it is the same as #410 atleast, and it is also only on 8.8.0, it is fine on the previous version 8.7.1. You can get that in releases here on github.

**Comment by hipersayanX** [comment]
@ktaragorn There are other things that needs to be fixed before releasing anything. Also, I'm busy with other things, so don't expect any release on the near time.

**Comment by ktaragorn** [comment]
> @ktaragorn There are other things that needs to be fixed before releasing anything. Also, I'm busy with other things, so don't expect any release on the near time.

:+1: was just letting anyone else facing the same know.

**Comment by bkw777** [comment]
Same here.
```
./webcamoid/webcamoid.sh 
QApplication: invalid style override passed, ignoring it.
    Available styles: Windows, Fusion
QQmlApplicationEngine failed to load component
qrc:/Webcamoid/share/qml/main.qml:20 plugin cannot be loaded for module "QtQuick": The plugin '/usr/lib/x86_64-linux-gnu/qt5/qml/QtQuick.2/libqtquick2plugin.so' uses incompatible Qt library. (5.15.0) [release]
```

**Comment by evanescente-ondine** [comment]
Facing the same now.
./webcamoid-8.8.0-x86_64.AppImage 
QQmlApplicationEngine failed to load component
qrc:/Webcamoid/share/qml/main.qml:20 plugin cannot be loaded for module "QtQuick": The plugin '/usr/lib/x86_64-linux-gnu/qt5/qml/QtQuick.2/libqtquick2plugin.so' uses incompatible Qt library. (5.15.0) [release]

Debian Bullseye. Didn't work with the stable version either.

**Comment by evanescente-ondine** [comment]
I'm fine with downgrading QT but to what version ?
text says **superior** or equal to 5.9.xx. Obviously not too superior.
I work with videos, I'm not a programmer and building 8.7.1 is a nightmare. Debian testing's is too old and an issue makes it unworkable.
Really starting to annoy me...
I think I have more chance with 8.8 and adding another QT library on top of my own.
So which one ?

Thanks...

**Comment by skorokithakis** [comment]
Same here, neither AppImage nor portable version works on Ubuntu 20.10.

**Comment by hipersayanX** [comment]
Problem solved in 9.0.0, closing.
